src/main/core/Event/Functional/ResourceEvaluationEvent.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Claroline Connect package.
+ *
+ * (c) Claroline Consortium <consortium@claroline.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Claroline\CoreBundle\Event\Functional;
+
+use Claroline\CoreBundle\Entity\Resource\ResourceEvaluation;
+use Claroline\CoreBundle\Entity\Resource\ResourceNode;
+use Claroline\CoreBundle\Entity\User;
+use Symfony\Contracts\EventDispatcher\Event;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ResourceEvaluationEvent extends Event
```

Comment:
An event already exist : `Claroline\CoreBundle\Event\Resource\EvaluateResourceEvent`. Must be removed.